src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -133,11 +134,10 @@ public void bindArmControls() {
 	}
 
 	public void bindIntakeControls() {
-		// CommandScheduler.getInstance()
-		// .setDefaultCommand(s.intakeSubsystem, new IntakeDefaultCommand(s.intakeSubsystem));
+		CommandScheduler.getInstance()
+				.setDefaultCommand(s.intakeSubsystem, new IntakeDefaultCommand(s.intakeSubsystem));
 
-		intakeInButton.onTrue(
-				new IntakeSetInCommand(s.intakeSubsystem).until(s.intakeSubsystem::isSecured));
+		intakeInButton.onTrue(new IntakeSetInCommand(s.intakeSubsystem));
```

Comment:
Now that the default command works (or should work) and the speeds aren't switched we should probably do something to make sure the `IntakeSetInCommand` will actually stop running (like maybe doing while the button is held down, or checking the distance sensor), otherwise the default command won't actually run after we intake a game object.